pyuoi/linear_model/elasticnet.py

```diff
@@ -49,20 +51,50 @@ def selection_lm(self):
         return self.__selection_lm
 
     def get_reg_params(self, X, y):
+        """Calculates the regularization parameters (alpha and lambda) to bbe
+        used for the provided data.
+
+        Note that the Elastic Net has two regularization parameters: alpha,
+        which specifies the ratio between the L1 and L2 penalties, and lambda,
```

Comment:
In this description, I don't understand what lambda is.

Reply:
Resolved in 20d6612.